src/Core/src/Handlers/ViewHandlerExtensions.Android.cs

```diff
@@ -116,10 +116,7 @@ internal static void PlatformArrangeHandler(this IViewHandler viewHandler, Rect
 				return;
 			}
 
-			var left = Context.ToPixels(frame.Left);
-			var top = Context.ToPixels(frame.Top);
-			var bottom = Context.ToPixels(frame.Bottom);
-			var right = Context.ToPixels(frame.Right);
+			var (left, top, right, bottom) = context.ToPixels(frame);
```

Comment:
Slightly impacts tests with Labels and custom FontFamily (around 1-2% of changes detected when checking the image).
![image](https://github.com/user-attachments/assets/a3d9fad4-d34f-4429-a2da-a382f6c021cb)

Example:
![image](https://github.com/user-attachments/assets/0af94a6f-bdc7-4676-9ec9-ad193ddcff42)

Failing tests:
- FontFamily
- FontFamilyLoadsDynamically
- VerifyTextDecorationAppliedProperly
- Bugzilla53834Test
- ChildFlexLayoutContentShouldAppear
- SystemFontsShouldRenderCorrectly
- LabelHyperlinkUnderlineColor

Could you review and update the failing ones? Let me know if can help with something.